magento/magento2
# magento/magento2#12396: Total Amount cart rule without tax

Added new condition type to give user opportunity to choose the configuration.




### Description (*)
I thought about the fix of what should be fixed in here. But I guess it's up to the user what to choose and what conditions to have to apply the rule. 
I decided to add new Cart Attribute to **Subtotal (Excl. Tax)** which is basically one of the totals _base_subtotal_with_discount_ 


### Fixed Issues (if relevant)

1. magento/magento2#<[12396](https://github.com/magento/magento2/issues/12396)>: "Total Amount" cart rule without tax

### Manual testing scenarios (*)

1. Go to the Admin Panel. Open Marketing->Cart Price Rules
2. Add new rule or edit existing one
3. Under the Condition tab you will be able to see new Configuration Subtotal (Excl. Tax).

### Contribution checklist (*)
